projects/sm-pwr/bop-cortix-test.ipynb

```diff
@@ -46,6 +46,33 @@
     "**11Dec2020**"
```

Comment:
<p>should be condenser = Condenser()</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dpploy/engy-4390/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>